doc/install.rst

```diff
@@ -124,6 +121,35 @@ Some dependencies can be specified as follows. Note only some of these support
   Path to Python used by :code:`sphinx-build`. Default: shebang of
   :code:`sphinx-build`.
 
+Some dependencies can be specified as follows. Note only some of these support
+:code:`--with-FOO=no`, as listed.
+
+Optional bind selection: :code:`--bind-FOO`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
I think these will want to be spelled `--enable-bind-FOO`, because `--enable` gets special treatment in Autotools.